### PR TITLE
Prevent longer than 20char tags

### PR DESCRIPTION
**Pull request by mstratford**
A dormant branch that I just found the code for. Not sure if we need to put season / timeslot stuff in too.

Mainly an issue because it looks / feels sensible. Seen some abuse that just looks a bit meh in the past.

### CODE REVIEW

**Comment by URY-GitHub** [comment]
Thank you for this pull request, @mstratford!

@SplashyFish14, would you mind reviewing this PR, please? Feel free to ask anyone for help if you're not quite sure what you're doing.

**Comment by mstratford** [comment]
Oh, hang on, this may have been re-done already.

File: src/Classes/ServiceAPI/MyRadio_Podcast.php [hunk 1]
```diff
@@ -403,6 +403,12 @@ public static function create(
         MyRadio_Show $show = null,
         $credits = null
     ) {
+        //Explode the tags
+        $tags = explode(' ', $tags);
```
Comment (markspolakovs) [inline src/Classes/ServiceAPI/MyRadio_Podcast.php:407]: Shouldn't this use CoreUtils::explodeTags?

Comment (mstratford) [reply src/Classes/ServiceAPI/MyRadio_Podcast.php:407]: I think coreUtils::explodeTags was created after this, and supersedes this PR, closing 👍

**Pull request closed without merging**